packages/donor/tsconfig.json

```diff
@@ -27,5 +27,12 @@
   },
   "include": [
     "src"
-  ]
+  ],
+  "ts-node": {
```

Comment:
Should this also be in `tsconfig.build.json`?

Reply:
I'll get back to you on this one,
Not really sure,
I manly use it to generate the signed apk file,
( Running npm scripts with ts)

Which is a dev script? 🤷🏻‍♂️

Reply:
it's ignoring the "extends" property.

that's the current config inheritance "tree":
```/tsconfig.build.json``` --> ```/tsconfig.json``` --> ```packages/donor/tsconfig.json```

I've tried moving the "ts-node" field to the other two files with no luck, the scripts that depend on it are failing.

Reply:
I am not super familiar with this, so I trust you.

If you need help here, contact Harel